projects/table-builder/src/lib/components/header-menu/header-menu.component.html

```diff
@@ -97,7 +97,7 @@
         </ng-container>
       </ng-template>
 
-    <button mat-button (click)="onEnter(myForm.value)" [disabled]="!myForm.value.filterValue && myForm.value.filterValue !== false" disableRipple>
+    <button mat-button (click)="onEnter(myForm.value)" [disabled]="myForm.value.filterValue == undefined" disableRipple>

```

Comment:
===

Reply:
NO. `==null` includes `null` and `undefined`. The one good use case for `==`